actors/builtin/miner/miner_actor.go

```diff
@@ -1753,8 +1755,10 @@ func handleProvingDeadline(rt Runtime) {
 			// Release pledge requirements for the sectors expiring on-time.
 			// Pledge for the sectors expiring early is retained to support the termination fee that will be assessed
 			// when the early termination is processed.
-			pledgeDelta = big.Sub(pledgeDelta, expired.OnTimePledge)
```

Comment:
I believe this was just subtracting from zero before to get a negative directly, but please double check

Reply:
Please revert. Intention is to accumulate a pledge delta over a number of different actions in this handler, in parallel with accumulating `penaltyTotal`. It just happens that this is the first and only contribution to the delta as the code stands now, but this is fragile if something affecting pledge is introduced further up.

Reply:
done